soc/arm/qemu_cortex_a53/mmu_regions.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Broadcom
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <soc.h>
+#include <arch/arm/aarch64/arm_mmu.h>
+
+static const struct arm_mmu_region mmu_regions[] = {
+
+	MMU_REGION_FLAT_ENTRY("GIC",
+			      0x8000000, 0x20000,
```

Comment:
Can we use `DT_INST_0_ARM_*` (and `DT_INST_0_ARM_PL011_*` for the UART) instead of hardcoding memory address and size?

Reply:
yes, will change.